cellprofiler/modules/relateobjects.py

```diff
@@ -22,8 +32,6 @@
 parent objects, you might want to expand the parent objects using
 **ExpandOrShrink** or **IdentifySecondaryObjects**.
 
-|
```

Comment:
This line is needed to ensure proper spacing between the table and the paragraph above.